<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>depositos</class>
 <widget class="QDialog" name="depositos">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>612</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Deposito Internacional</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>371</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../banco.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>30</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
color: rgb(187, 148, 91);</string>
   </property>
   <property name="text">
    <string>Depósito Bancario Internacional</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="margin">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Tipo de Documento *</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbDocumento">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <item>
    <property name="text">
     <string>---------Seleccione una Opción---------</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DNI - Documento Nacional de Identidad</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PSS - Pasaporte</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>VISA</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>150</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Número de Documento *</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtDoc">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>180</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
  </widget>
  <widget class="QComboBox" name="cbGiro">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>500</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <item>
    <property name="text">
     <string>---------Seleccione una Opción---------</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Honorarios/Servicios Técnicos</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Remesas/Giros</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Donaciones</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>470</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Motivo del Giro *</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>470</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Monto en Dolares (USD) *</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtMonto">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>500</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkTerminos">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>570</y>
     <width>261</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Aceptar Terminos y Condiciones *</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRegis">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>560</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(34, 91, 78);
color: rgb(255, 255, 255);
border-radius:8px;</string>
   </property>
   <property name="text">
    <string>Registrar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtPrimerNombre">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Primer Nombre *</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>230</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Segundo Nombre</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtSegundoNombre">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>260</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtSegundoApellido">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>340</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Primer Apellido *</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>310</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Segundo Apellido</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtPrimerApellido">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>390</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Género *</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbGenero">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>420</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <item>
    <property name="text">
     <string>---------Seleccione una Opción---------</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Femenino</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Masculino</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Otro</string>
    </property>
   </item>
  </widget>
  <widget class="QDateEdit" name="txtFecha">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>420</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>390</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Fecha de Nacimiento</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>390</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Lugar de Nacimiento *</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbLugar">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>420</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <item>
    <property name="text">
     <string>---------Seleccione una Opción---------</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 90, 90);</string>
   </property>
   <property name="text">
    <string>Los campos con * son obligatorios</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>791</width>
     <height>2</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>2</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(234, 219, 188);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
